src/Scoreboard.h

```diff
@@ -52,11 +58,32 @@ class cObjective
 		otStatEntityKilledBy
```

Comment:
Since the enum has been renamed, the prefix should be changed, too (the only argument against this would be when it is exported to the API, but we agreed that the API can change safely)

Reply:
Changed to "cr..."